runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsPlugin.java

```diff
@@ -190,26 +190,26 @@ public void onSetMessage(SetMessage setMessage)
 		// being reused
 		messageNode.setRuneLiteFormatMessage(null);
 
-		if (config.lvl() && message.toLowerCase().equals(TOTAL_LEVEL_COMMAND_STRING))
+		if (config.lvl() && message.toLowerCase().matches(TOTAL_LEVEL_COMMAND_STRING))
```

Comment:
All of these should be using `PATTERN.matcher(string).find()` to match substrings. That way, the regexes defined need only define the start of the string they intend to match.